### PR TITLE
Update default Sigdep3 database

### PR DESCRIPTION
**Pull request by HerbertYiga**
Updating default Sigdep3 database

### CODE REVIEW

File: configs/sigdep3/db/update_default_db.sql [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        0,
+        NULL,
+        NULL,
+        NULL,
+        NULL,
+        NULL,
+        'd5efc10e-bf9c-4813-96ef-2969cec85cbb'
+    );
+
+INSERT INTO `fhir_patient_identifier_system` (
```
Comment (pmanko) [inline configs/sigdep3/db/update_default_db.sql:35]: We can just collapse this into a single INSERT statement.

**Review by pmanko — CHANGES_REQUESTED** [review]
@HerbertYiga I see a possible barrier to the update working as intended. The table that we're inserting into might not exist yet, since the structure is not part of the initial SQL, and the table gets created from liquibase in the fhir2 module. You might have to add a `CREATE TABLE` command in the script before `INSERT`ing the data.

File: configs/sigdep3/db/update_default_db.sql (hunk 1 above)
Comment (HerbertYiga) [reply configs/sigdep3/db/update_default_db.sql:35]: sure

**Comment by HerbertYiga** [comment]
> I see a possible barrier to the update working as intended. The table that we're inserting into might not exist yet, since the structure is not part of the initial SQL, and the table gets created from liquibase in the fhir2 module. You might have to add a `CREATE TABLE` command in the script before `INSERT`ing the data.

quickly double checking this!

**Comment by mozzy11** [comment]
I think , we first need to run the Sigdep3 instance with the FHIR2 module , and do a fresh DB dump with all the FHIR2 changesets loaded.
That way we can always add fhir system URL via a separate script

**Comment by HerbertYiga** [comment]
the script here works well  after testing on the server,Hi @pmanko ,  just want to be sure of some other identifier that we need to add,the script here adds Art Id and Upi only

File: configs/sigdep3/db/update_identifier.sql [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  CONSTRAINT `fhir_patient_identifier_system_patient_identifier_type_fk` FOREIGN KEY (`patient_identifier_type_id`) REFERENCES `patient_identifier_type` (`patient_identifier_type_id`),
+  CONSTRAINT `fhir_patient_identifier_system_retired_by_fk` FOREIGN KEY (`retired_by`) REFERENCES `users` (`user_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+INSERT INTO `fhir_patient_identifier_system` (`fhir_patient_identifier_system_id`,`patient_identifier_type_id`,`url`,`name`,`description`,`creator`,`date_created`,`retired`,`retired_by`,`date_retired`,`retire_reason`,`changed_by`,`date_changed`,`uuid`)
+VALUES ('1','3','http://clientregistry.org/artnumber','ART Number','ART Number',1,'2013-12-27 00:00:00',0,NULL,NULL,NULL,NULL,NULL,'d5efc10e-bf9c-4813-96ef-2969cec85cbb'),
+    ('2','6','http://cote.divore/ext/identifier/UPI','UPI','UPI',1,'2013-12-27 00:00:00',0,NULL,NULL,NULL,NULL,NULL,'b4a56d59-42b2-4a4a-86cc-0fb3188bca11'),('3','1','http://tap-cdi.openelis.org/ext/identifier/openmrs-id','openmrs-id','openmrs-id',1,'2013-12-27 00:00:00',0,NULL,NULL,NULL,NULL,NULL,'17b7af60-bc69-4c1c-867e-762219f2ab19');
```
Comment (HerbertYiga) [inline configs/sigdep3/db/update_identifier.sql:37]: Hi @pmanko your updated script here https://github.com/I-TECH-UW/ci-hie-demo/issues/11 had a repetition for the UPI in place of the openmrs Id, so i insert the openmrs id through this 

('3','1','http://tap-cdi.openelis.org/ext/identifier/openmrs-id','openmrs-id','openmrs-id',1,'2013-12-27 00:00:00',0,NULL,NULL,NULL,NULL,NULL,'17b7af60-bc69-4c1c-867e-762219f2ab19')

**Pull request merged**